__init__.py

```diff
@@ -51,6 +51,15 @@ async def async_on_node_added(node):
                 hass.async_create_task(discovery.async_load_platform(
                     hass, Platform.LIGHT, DOMAIN, discovery_info, config))
                 pass
+            elif "OccupancySensing" in endpoint.clusters:
+                if "OnOff" in endpoint.clusters:
```

Comment:
I am thinking if we should do this without checking if Occupancy cluster is available.

Reply:
Many devices report OnOff in SupportedGeneratedCommands, this would require mapping to relevant BinarySensor HA DeviceTypes